build_scripts/windows/scripts/build_saw_zip.cmd

```diff
@@ -0,0 +1,239 @@
+
+if "%CLI_VERSION%"=="" (
+    echo Please set the CLI_VERSION environment variable, e.g. 2.0.13
+    goto ERROR
+)
+set PYTHON_VERSION=3.10.5
+set SPYTHON_VERSION=3.10.22165.3
+
+set WIX_DOWNLOAD_URL="https://azurecliprod.blob.core.windows.net/msi/wix310-binaries-mirror.zip"
+@REM windows-http only support amd64
+set PYTHON_DOWNLOAD_URL="https://www.python.org/ftp/python/%PYTHON_VERSION%/python-%PYTHON_VERSION%-embed-amd64.zip"
+set NUGET_DOWNLOAD_URL="https://dist.nuget.org/win-x86-commandline/v6.2.1/nuget.exe"
+@REM TODO download from storage account
+set SPYTHON_SOURCE="C:\Users\hanglei"
+set WINDOWS_HTTP_PATH="C:\Users\hanglei\Downloads\windows_http-0.22.203.1-cp310-cp310-win_amd64.whl"
+
+REM https://pip.pypa.io/en/stable/installation/#get-pip-py
+set GET_PIP_DOWNLOAD_URL="https://bootstrap.pypa.io/get-pip.py"
+
+REM Set up the output directory and temp. directories
+echo Cleaning previous build artifacts...
+set OUTPUT_DIR=%~dp0..\out
+if exist %OUTPUT_DIR% rmdir /s /q %OUTPUT_DIR%
+mkdir %OUTPUT_DIR%
+
+set ARTIFACTS_DIR=%~dp0..\artifacts
+mkdir %ARTIFACTS_DIR%
+set TEMP_SCRATCH_FOLDER=%ARTIFACTS_DIR%\cli_scratch
+set BUILDING_DIR=%ARTIFACTS_DIR%\azure-cli
+set WIX_DIR=%ARTIFACTS_DIR%\wix
+set PYTHON_DIR=%ARTIFACTS_DIR%\Python
+set SPYTHON_DIR=%ARTIFACTS_DIR%\SPython
+set SPYTHON_EXE=%SPYTHON_DIR%\Microsoft.Internal.SPython.win32.%SPYTHON_VERSION%\tools\python.exe
+set SPYTHON_LIB=%SPYTHON_DIR%\Microsoft.Internal.SPython.win32.%SPYTHON_VERSION%\tools\Lib
+set NUGET_DIR=%ARTIFACTS_DIR%\Nuget
+set PROPAGATE_ENV_CHANGE_DIR=%~dp0..\propagate_env_change
+
+set REPO_ROOT=%~dp0..\..\..
+
+REM reset nuget dir
+if not exist %NUGET_DIR% (
+    mkdir %NUGET_DIR%
+    pushd %NUGET_DIR%
+    curl %NUGET_DOWNLOAD_URL%
+    popd
+)
+
+REM reset spython dir
+if exist %SPYTHON_DIR% rmdir /s /q %SPYTHON_DIR%
+if not exist %SPYTHON_DIR% (
+    mkdir %SPYTHON_DIR%
+    cd %SPYTHON_DIR%
+    %NUGET_DIR%\nuget.exe install Microsoft.Internal.SPython.win32 -version %SPYTHON_VERSION% -Source %SPYTHON_SOURCE%
+)
+
+@REM exit /b
+
+REM reset working folders
+if exist %BUILDING_DIR% rmdir /s /q %BUILDING_DIR%
+REM rmdir always returns 0, so check folder's existence
+if exist %BUILDING_DIR% (
+    echo Failed to delete %BUILDING_DIR%.
+    goto ERROR
+)
+mkdir %BUILDING_DIR%
+
+if exist %TEMP_SCRATCH_FOLDER% rmdir /s /q %TEMP_SCRATCH_FOLDER%
+if exist %TEMP_SCRATCH_FOLDER% (
+    echo Failed to delete %TEMP_SCRATCH_FOLDER%.
+    goto ERROR
+)
+mkdir %TEMP_SCRATCH_FOLDER%
+
+if exist %REPO_ROOT%\privates (
+    copy %REPO_ROOT%\privates\*.whl %TEMP_SCRATCH_FOLDER%
+)
+
+REM ensure wix is available
+if exist %WIX_DIR% (
+    echo Using existing Wix at %WIX_DIR%
+)
+if not exist %WIX_DIR% (
+    mkdir %WIX_DIR%
+    pushd %WIX_DIR%
+    echo Downloading Wix.
+    curl --output wix-archive.zip %WIX_DOWNLOAD_URL%
+    unzip wix-archive.zip
+    if %errorlevel% neq 0 goto ERROR
+    del wix-archive.zip
+    echo Wix downloaded and extracted successfully.
+    popd
+)
```

Comment:
WiX is no longer necessary if we build a ZIP. Better to confirm with SAW team if ZIP is accepted.